Tests/MapboxMapsTests/Snapshot/MapboxMapsSnapshotTests.swift

```diff
@@ -1,10 +1,134 @@
 import XCTest
+import MapboxMaps
+import MapboxCoreMaps
+import Turf
+import Foundation
+import UIKit
 #if canImport(MapboxMaps)
 @testable import MapboxMaps
 #else
 @testable import MapboxMapsSnapshot
 #endif
-
 //swiftlint:disable explicit_top_level_acl explicit_acl
-class MapboxMapsSnapshotTests: XCTestCase {
+class MapboxMapsSnapshotTests: IntegrationTestCase {
+   
+    var geoJSONSource = GeoJSONSource()
+    var snapshotView = UIImageView()
+    var blankAttachment: XCTAttachment!
+    var newAttachment: XCTAttachment!
+    
+    // Create snapshot options
+    private func snapshotterOptions(size: CGSize) -> MapSnapshotOptions {
+        let resourceOptions = ResourceOptions(accessToken: accessToken)
+        let options = MapSnapshotOptions(size: size, resourceOptions: resourceOptions, pixelRatio: 4)
+        return options
+    }
+    
+    // Testing creating the snapshot
+    func CapturingSnapshotterInSnapshotCompletion() {
+        let timeout: TimeInterval = 10.0
+        let expectation = self.expectation(description: "snapshot")
+        let options = snapshotterOptions(size: CGSize(width: 300, height: 300))
+        DispatchQueue.main.async {
+            let snapshotter = Snapshotter(options: options)
+            snapshotter.camera = CameraOptions(center: CLLocationCoordinate2D(latitude: 38.9180379, longitude: -77.0600235),zoom: 5)
+            snapshotter.style.styleURL = StyleURL.light
```

Comment:
`StyleURL` has been renamed to `StyleURI`. You'll need to merge or rebase with `main`.